lib/everypolitician/pull_request/report/memberships.rb

```diff
@@ -13,11 +13,16 @@ def removed
         private
 
         def before_h
-          @before_h ||= before.memberships.map { |m| [m.document, m] }.to_h
+          @before_h ||= before.memberships.map { |m| [document_without_sources(m.document), m] }.to_h
         end
 
         def after_h
-          @after_h ||= after.memberships.map { |m| [m.document, m] }.to_h
+          @after_h ||= after.memberships.map { |m| [document_without_sources(m.document), m] }.to_h
+        end
+
+        def document_without_sources(doc)
+          doc.delete(:sources)
+          doc
```

Comment:
This doesn't feel like a very maintainable approach. Extending this to also ignore some other field(s) seems like something that's likely enough that we shouldn't need to rename a method (or chain a lot of similar methods together). We don't really want to over-engineer this either, so I'd be tempted to simply extend the document here with `Hash#except` method from ActiveSupport (or a suitable clone thereof) instead.